protocols/floodsub/src/rpc.proto

```diff
@@ -8,15 +8,15 @@ message RPC {
 
 	message SubOpts {
 		optional bool subscribe = 1; // subscribe or unsubcribe
-		optional string topicid = 2;
+		optional string topic_id = 2;
 	}
 }
 
 message Message {
 	optional bytes from = 1;
 	optional bytes data = 2;
 	optional bytes seqno = 3;
-	repeated string topicIDs = 4;
+	repeated string topic_ids = 4;
```

Comment:
Does prost not support camelcase? :thinking:

Reply:
Maybe there is a tweak but IIRC `topicIDs` was turned into `topic_i_ds`. I suppose `topicIds` would have worked too.

Reply:
Is this change backwards compatible with go-ipfs floodsub?

Reply:
Yes, as far as protocol buffers is concerned, names do not matter, hence those index numbers.

Reply:
Right, completely forgot about that. Sorry about the stupid question.